### PR TITLE
fix: skip instead of hard fail

### PR DESCRIPTION
**Pull request by caarlos0**
This makes partially set up brews, krews, etc skip when no archives match, instead of hard-failing.

I'm not sure if this is better or not, but it would make sense for things like partial builds (e.g. build for linux only), so the config might get reused.

Moreover, IMHO, its better to skip than hard fail because people will likely have to create a new tag anyway, due to go mod caching et al... so with this at least they don't get a broken release.

tl;dr: still thinking about it.

### CODE REVIEW

**Comment by caarlos0** [comment]
will leave this out for now
